### ISSUE TITLE
Video not display

### ISSUE THREAD

**Issue by ThanhDeveloper** [padymies/angular-webrtc-sample#1]
My webcam is display led but not show camera. Please check your resource 
![image](https://user-images.githubusercontent.com/48196420/160518382-935f0164-991a-4d95-8cd2-214944397443.png)

**Comment by TamilJackson** [comment]
Yes video not displaying. Showing error after clicking "Init Call" button 

1.ERROR Error: Uncaught (in promise): InvalidStateError: Failed to execute 'addIceCandidate' on 'RTCPeerConnection': The remote description was null
Error: Failed to execute 'addIceCandidate' on 'RTCPeerConnection': The remote description was null

2.ERROR Error: Uncaught (in promise): InvalidStateError: Failed to execute 'setRemoteDescription' on 'RTCPeerConnection': Failed to set remote answer sdp: Called in wrong state: stable
Error: Failed to execute 'setRemoteDescription' on 'RTCPeerConnection': Failed to set remote answer sdp: Called in wrong state: stable

**Comment by Tsquared86** [comment]
I have the same issue
